python/ray/tests/test_cancel.py

```diff
@@ -92,6 +96,179 @@ def dummy(a: SlowToDeserialize):
         ray.get(obj)
 
 
+def test_defer_sigint():
```

Comment:
does this test work on Windows?

Reply:
Hmm apparently not, I'll skip these for Windows.

Reply:
Changed from `os.kill()` to `_thread.interrupt_main()`, since it appears that the latter will propagate the SIGINT signal on all platforms -while the former will not on Windows: https://bugs.python.org/msg339528

Let's see if that fixes the Windows build.

Reply:
Windows build was fixed with this change.